.github/workflows/ci.yml

```diff
@@ -135,11 +135,11 @@ jobs:
         continue-on-error: true
       - name: Upload coverage results to Codecov
         if: matrix.coverage
-        uses: codecov/codecov-action@v1.0.3
+        uses: codecov/codecov-action@v1
         with:
           name: phpunit-php${{ matrix.php }}
           flags: phpunit
-          token: ${{ secrets.CODECOV_TOKEN }}
+          fail_ci_if_error: true
```

Comment:
We want the step to fail, but not the job to fail (`continue-on-error: true` below)